src/components/Layout/index.tsx

```diff
@@ -33,43 +34,16 @@ const Layout: FC<Props> = ({ children }) => {
             <NavBar />
             <div className={styles.wrapper}>
                 <div className={styles.header}>
-                    {navBarContent('Tasks', '/', router.pathname === '/')}|
-                    {navBarContent(
-                        'Issues',
-                        '/issues',
-                        router.pathname === '/issues'
-                    )}
-                    |
-                    {navBarContent(
-                        'Mine',
-                        '/mine',
-                        router.pathname === '/mine'
-                    )}
-                    {/* TODO: Uncomment when DS(Chanllenges) is ready */}
-                    {/* |
-                    {navBarContent(
-                        'DS',
-                        '/challenges',
-                        router.pathname === '/challenges'
-                    )} */}
-                    |
-                    {navBarContent(
-                        'Open PRs',
-                        '/openPRs',
-                        router.pathname === '/openPRs'
-                    )}
-                    |
-                    {navBarContent(
-                        'Stale PRs',
-                        '/stale-pr',
-                        router.pathname === '/stale-pr'
-                    )}
-                    |
-                    {navBarContent(
-                        'Idle Users',
-                        '/idle-users',
-                        router.pathname === '/idle-users'
-                    )}
+                    {navBarContentMock.map((element, index) => (
+                        <React.Fragment key={index}>
+                            {navBarContent(
+                                element.title,
+                                element.refURL,
+                                router.pathname === element.pathName
+                            )}
+                            {element.pipeSymbol}
+                        </React.Fragment>
+                    ))}
```

Comment:
Why is the availability panel excluded from it?

Reply:
what does this availability panel mean?

Reply:
availability panel is behind dev flag which is handled separately